bin/list-all

```diff
@@ -12,8 +12,16 @@ set -o nounset -o pipefail -o errexit
 # shellcheck source=../lib/utils.sh
 source "$(dirname "$0")/../lib/utils.sh"
 
+function print_only_fully_numeric_items() {
+  local version_numbers="$1"
+  grep -E '^[0-9.]+$' <<< "$version_numbers"
+}
+
+asdf_nodejs_bin_dir="$(dirname "$0")"
+all_definitions_from_node_build="$("${asdf_nodejs_bin_dir}/../.node-build/bin/node-build" --definitions)"
```

Comment:
We have and wrapper script around node-build to make it easier to run it with the necessary variables, check the install script for [an example](https://github.com/asdf-vm/asdf-nodejs/blob/364c0783408e25130c254abfaaec4747404c8f5e/bin/install#L37)

```suggestion
all_definitions_from_node_build="$("$ASDF_NODEJS_PLUGIN_DIR/lib/commands/command-nodebuild.bash"--definitions)"
```